scripts/mixins/families/uragnite.lua

```diff
@@ -56,16 +56,31 @@ local function exitShell(mob)
     mob:setMobMod(xi.mobMod.NO_MOVE, 0)
 end
 
+local function shellTimer(mob)
+    -- Uragnites go into their shell every 40-60 seconds
```

Comment:
I always thought it was "big hit" like anything that took a large chunk of hp in 1 go be it a crit or a ws to make them go in, then timed for them to come out again.

Reply:
It is both. If you see farther down, I still have code that they are forced into their shell from damage. In retail, you can tell the difference between the two, as when they are forced into their shell through damage they react with a venom shell every time. But normal uragnite mobs, even when they're just wandering unclaimed you can watch them go into their shell and out of it.

Reply:
I also tested the damage portion, it not a huge hit because I was able to get them forced into their shell by just tickling them with a lvl 1 weapon until I hit 10% health and they would go into their shell.

Reply:
From my current test, the timer seems to go from 25s to 50s while engaged and left alone.
Granted, this is a very limited test.